src/dataflow/environments/resolve-by-name.ts

```diff
@@ -105,3 +110,88 @@ export function resolveToConstants(name: Identifier | undefined, environment: RE
 		from:  def.type
 	}));
 }
+
+
+export function getAliases(sourceIds: readonly NodeId[], dataflow: DataflowGraph, environment: REnvironmentInformation): NodeId[] | undefined {
```

Comment:
In general it would be nice to have a config option whether to use getAlias or resolves to built in constant. Can you explore this within this PR? (or defer as next step after this issue?)

Reply:
This switch is probably best implemented by abstracting the call to resolve, ideally we add a flag to the config format which then configure an appropriate handler/resolve object on startup